### PR TITLE
Change variable name

### PR DESCRIPTION
**Pull request by andrewheiss**
The weight column is named `wt` in `mtcars`

I reinstalled keras+tensorflow so many times to get the example to run and it WOULDN'T WORK, but all other examples in the regular keras and tensorflow libraries did, but it works just fine with `wt` (as expected), so \\(•◡•)/

I just have no idea what all the model output means yet :)

### CODE REVIEW

**Comment by rdrr1990** [comment]
thanks! (I usually just write `mpg ~ .` and suppose I should stick to that. Apologies for any inconvenience!

**Comment by andrewheiss** [comment]
Ha, not a problem! Now I know a lot more about the virtualenvs that `install_keras()` and `install_tensorflow()` make :)

**Comment by rdrr1990** [comment]
actually very good--if you want to take advantage of GPU on a server, you typically have to dig into that stuff...
